py/selenium/webdriver/common/service.py

```diff
@@ -87,35 +88,21 @@ def start(self) -> None:
            or when it can't connect to the service
         """
         try:
-            cmd = [self.path]
-            cmd.extend(self.command_line_args())
-            self.process = subprocess.Popen(
-                cmd,
-                env=self.env,
-                close_fds=system() != "Windows",
-                stdout=self.log_file,
-                stderr=self.log_file,
-                stdin=PIPE,
-                creationflags=self.creation_flags,
-            )
-            log.debug(f"Started executable: `{self.path}` in a child process with pid: {self.process.pid}")
-        except TypeError:
-            raise
-        except OSError as err:
-            if err.errno == errno.ENOENT:
-                raise WebDriverException(
-                    f"'{os.path.basename(self.path)}' executable needs to be in PATH. {self.start_error_message}"
-                )
-            elif err.errno == errno.EACCES:
-                raise WebDriverException(
-                    f"'{os.path.basename(self.path)}' executable may have wrong permissions. {self.start_error_message}"
-                )
-            else:
-                raise
-        except Exception as e:
-            raise WebDriverException(
-                f"The executable {os.path.basename(self.path)} needs to be available in the path. {self.start_error_message}\n{str(e)}"
-            )
+            self._start_process(self.path)
+        except WebDriverException as err:
+            if "executable needs to be in PATH" in err.msg:
+                logger.info("driver not found in PATH, trying Selenium Manager")
+                browser = self.__class__.__module__.split(".")[-2]
+                try:
+                    path = SeleniumManager.driver_location(browser)
+                except OSError as new_err:
+                    if new_err.errno == errno.ENOENT:
+                        raise err
+                    else:
+                        raise new_err
```

Comment:
minor: `raise new_error from None` here? do we care about the previous exc info?

Reply:
Yeah, I want to throw the error that chromedriver is not found, not the error that selenium manager isn't found. For right now Selenium Manager isn't something the user can/should do anything about. The fix is for them to download chromedriver or use a manager. Maybe we can link to the docs for these, too.